src/components/SourcesTree.js

```diff
@@ -119,6 +121,27 @@ let SourcesTree = React.createClass({
     return Svg("folder");
   },
 
+  onContextMenu(event, item) {
+    event.stopPropagation();
+    event.preventDefault();
+
+    const source = item.path;
+    const copySourceUrl = {
+      id: "node-menu-copy-source",
+      label: "Copy Source URL",
+      accesskey: "X",
+      disabled: false,
+      click: () => copyToTheClipboard(source)
```

Comment:
Not sure how to fix this, the formatting of item.path is different from the others but it's probably not safe to just format item.path?